src/pkg/build/conveyorPacker_docker_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package build
+
+import (
+	"testing"
+
+	"github.com/singularityware/singularity/src/pkg/sylog"
+)
+
+// TestPull tests if we can pull an ubuntu image from dockerhub
+func TestConveyor(t *testing.T) {
+	def, err := NewDefinitionFromURI("docker://centos")
+	if err != nil {
+		sylog.Fatalf("unable to parse URI docker://ubuntu:18.04: %v\n", err)
+	}
+
+	dc := &DockerConveyor{}
+
+	if err := dc.Get(def); err != nil {
+		t.Fatal("failed to pull:", err)
+	}
+}
+
+// TestFurnish checks if we can create a Kitchen
+func TestPacker(t *testing.T) {
+	def, err := NewDefinitionFromURI("docker://ubuntu:18.04")
```

Comment:
While I think about it... this is probably a poor choice for a default test image, as it will not run on Red Hat / CentOS 6 as their kernel is too old. A debian or older Ubuntu may be better? Or maybe alpine.... it's much smaller, and we are just trying to test the basic functionality here... ubuntu can be tested in integration tests.